examples/animation/double_pendulum_animated.py

```diff
@@ -53,38 +59,46 @@ def derivs(state, t):
 # integrate your ODE using scipy.integrate.
 y = integrate.odeint(derivs, state, t)
 
-x1 = L1*sin(y[:, 0])
-y1 = -L1*cos(y[:, 0])
+x1 = length1*np.sin(y[:, 0])
+y1 = -length1*np.cos(y[:, 0])
 
-x2 = L2*sin(y[:, 2]) + x1
-y2 = -L2*cos(y[:, 2]) + y1
+x2 = length2*np.sin(y[:, 2]) + x1
+y2 = -length2*np.cos(y[:, 2]) + y1
 
 fig = plt.figure()
-ax = fig.add_subplot(111, autoscale_on=False, xlim=(-2, 2), ylim=(-2, 2))
+ax = fig.add_subplot(111, autoscale_on = False, xlim = (-2, 2), ylim = (-2, 2))
 ax.set_aspect('equal')
 ax.grid()
 
-line, = ax.plot([], [], 'o-', lw=2)
+line, = ax.plot([], [], 'o-', lw = 2)
 time_template = 'time = %.1fs'
 time_text = ax.text(0.05, 0.9, '', transform=ax.transAxes)
 
 
 def init():
+    """Create a line and text to show time of the animation."""
     line.set_data([], [])
     time_text.set_text('')
+    plt.xlabel('horizontal position (m)')
+    plt.ylabel('vertical position (m)')
     return line, time_text
 
 
 def animate(i):
+    """Update the animation.
+
+    Updates the positions of the double pendulum and the time shown,
+    indexed by i.
+    """
     thisx = [0, x1[i], x2[i]]
     thisy = [0, y1[i], y2[i]]
 
     line.set_data(thisx, thisy)
-    time_text.set_text(time_template % (i*dt))
+    time_text.set_text(time_template % (i * dt))
     return line, time_text
 
 ani = animation.FuncAnimation(fig, animate, np.arange(1, len(y)),
-                              interval=25, blit=True, init_func=init)
+                              interval = 25, blit = True, init_func = init)
```

Comment:
Sorry, forgot to run the pep8 checker before pushing. I've amended the commit, and it looks like it passes now.